src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Profile Pages</title>
+	<link rel="stylesheet" href="/css/main.css" type"text/css">
+
+	<style>
+		#chat {
+			background-color: white;
+			height: 500px;
+			overflow-y: scroll
+		}
+	</style>
+
```

Comment:
Lines 4-13 seem to be indented a little bit too far in. Could you shift it so it matches the level of indentation for lines 15+?